api/can_api_v2_definition.py

```diff
@@ -532,6 +532,10 @@ class RegionSummary(base_model.APIBaseModel):
         ...,
         description="3 digit Health Service Area identification code. For CBSA, state, and country regions hsa is omitted. For more on HSAs see: https://github.com/covid-projections/covid-data-model/blob/main/data/misc/README.md",
     )
+    hsaName: Optional[str] = pydantic.Field(
+        ...,
+        description="Name of Health Service Area. For CBSA, state, and country regions hsa is omitted. For more on HSAs see: https://github.com/covid-projections/covid-data-model/blob/main/data/misc/README.md",
```

Comment:
"hsa is omitted" => "hsaName is omitted" ?

Reply:
ah, thanks for the catch--updated in 9aeffdab7a5a35dcab0e502b1f7f5cd9ccc57eb2